openapi/components/schemas/Tag.yaml

```diff
@@ -0,0 +1,7 @@
+title: Tag
+type: string
+description: |-
+  Tags are a way to grant various access, assign restrictions or other kinds of metadata to various to objects such as worlds, users and avatars.
+
+  System tags starting with `system_` are granted automatically by the system, while admin tags with `admin_` are granted manually. More prefixes such as `language_ ` exists as well.
+x-internal: true
```

Comment:
what's this represent? ``x-internal``?

Reply:
`x-internal` is a vendor tag. Setting something as internal hides it from some doc exports, although it something usually put on endpoints, not models. A good example of  `x-internal` is putting in on a work-in-progress endpoint like `/licenseGroups` which currently just returns an array of nulls. Removed the tags from models.